codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionTestKitGenerator.scala

```diff
@@ -74,37 +74,37 @@ object ActionTestKitGenerator {
     val testKitClassName = s"${className}TestKit"
 
     s"""package ${service.fqn.parent.javaPackage};
-          |
-          |$imports
-          |
-          |$managedComment
-          |
-          |public final class $testKitClassName {
-          |
-          |  private Function<ActionCreationContext, $className> actionFactory;
-          |
-          |  private $className createAction() {
-          |    $className action = actionFactory.apply(new StubActionCreationContext());
-          |    action._internalSetActionContext(Optional.of(new StubActionContext()));
-          |    return action;
-          |  };
-          |
-          |  public static $testKitClassName of(Function<ActionCreationContext, $className> actionFactory) {
-          |    return new $testKitClassName(actionFactory);
-          |  }
-          |
-          |  private $testKitClassName(Function<ActionCreationContext, $className> actionFactory) {
-          |    this.actionFactory = actionFactory;
-          |  }
-          |
-          |  private <E> ActionResult<E> interpretEffects(Action.Effect<E> effect) {
-          |    return new ActionResultImpl(effect);
-          |  }
-          |
-          |  ${Format.indent(generateServices(service), 2)}
-          |
-          |}
-          |""".stripMargin
+        |
```

Comment:
just because I want to see the `|` aligned with the `p`. :-)

Reply:
shouldn't it align just *before* the `p`? :laughing:

Reply:
why not add the `|` at the first line then

Reply:
I will do it. I'm trying to hold a balance here and not look like a total formatting freak. :-)